wolfcrypt/src/hmac.c

```diff
@@ -99,7 +99,7 @@
     }
     int wolfSSL_GetHmacMaxSize(void)
     {
-        return CyaSSL_GetHmacMaxSize();
+        return WC_MAX_DIGEST_SIZE;
```

Comment:
This one is odd, but I reviewed the code and this is the right change.

Reply:
Yea, it surprised me as well. At first it seemed like a cyclical reference. This line is also what triggered the question about removal of FIPS_V1 stuff. Since we're removing it, this block of code will go away anyway.